helm/afm/files/conf.yaml

```diff
@@ -1,7 +1,7 @@
-app-uuid: {{ get .Values.labels "app.fybrik.io/app-uuid" | default "app-uuid-missing" }}
-{{ if .Values.read -}}
+app-uuid: {{ .Values.uuid | default "app-uuid-missing" }}
```

Comment:
Values.sample.yaml should be updated as well with this change.

Reply:
Updated.